builder/builder.go

```diff
@@ -61,8 +63,8 @@ type orderTOML struct {
 	Order dist.Order `toml:"order"`
 }
 
-// GetBuilder constructs builder from builder image
-func GetBuilder(img imgutil.Image) (*Builder, error) {
+// Get constructs a builder from a builder image
+func Get(img imgutil.Image) (*Builder, error) {
```

Comment:
These naming refactors seem super helpful, and generally I am in favor of them. I do wonder, however, whether they make sense as part of this PR, or if they should be added through a separate one, given they don't directly contribute to validating our mixins

Reply:
Will definitely keep in mind for next PR